addons/GME/Scripts/Game/GME/Editor/Components/Editor/SCR_PlacingEditorComponent.c

```diff
@@ -40,8 +40,29 @@ modded class SCR_PlacingEditorComponent : SCR_BaseEditorComponent
 	protected void GME_OnPlaceEntityServer(RplId prefabID, SCR_EditableEntityComponent entity, int playerID)
 	{
 		GME_Modules_Base module = GME_Modules_Base.Cast(entity.GetOwner());
+		// Handle placement of modules
 		if (module)
+		{
 			GME_OnModulePlacedServer(module, playerID);
+		}
+		// Put wheeled vehicles in first gear to prevent them sliding down slopes
+		else if (SCR_EditableVehicleComponent.Cast(entity))
+		{
+			Vehicle vehicle = Vehicle.Cast(entity.GetOwner());
+			if (!vehicle)
+				return;
+			
+			CarControllerComponent controller = CarControllerComponent.Cast(vehicle.GetVehicleController());
+			if (!controller)
+				return;
+			
+			VehicleWheeledSimulation simulation = controller.GetSimulation();
+			if (!simulation)
+				return;
+			
+			// Gear mapping: 0 => R, 1 => N, 2 => D/1, ...
+			simulation.SetGear(2);
```

Comment:
Any particular reason for gear over handbrake? 

```
	//! Sets brake inputs
	//! \param in should be in range < 0, 1 >
	//! \param hb indicating whether handbrake should be applied
	proto external void SetBreak(float in, bool hb);
	```
	
	I don't know how handbrake behaviour is implemented and if there might be vehicles without handbrakes, thus gear is better?

Reply:
Handbrake is actually already engaged when they spawn, but is ineffective. See https://feedback.bistudio.com/T175877.

Reply:
ah gotcha. Gear seems like a fix then.